Clean up script

Lint of `main.py`:

```
./main.py:11:1: W191 indentation contains tabs
./main.py:11:2: E117 over-indented
./main.py:12:1: W191 indentation contains tabs
./main.py:13:1: W191 indentation contains tabs
./main.py:14:1: W191 indentation contains tabs
./main.py:18:1: W191 indentation contains tabs
./main.py:18:2: E117 over-indented
./main.py:19:1: W191 indentation contains tabs
./main.py:19:3: E117 over-indented
./main.py:20:1: W191 indentation contains tabs
./main.py:21:1: W191 indentation contains tabs
./main.py:21:3: E117 over-indented
./main.py:22:1: W191 indentation contains tabs
./main.py:26:1: W191 indentation contains tabs
./main.py:26:2: E117 over-indented
./main.py:27:1: W191 indentation contains tabs
./main.py:31:1: W191 indentation contains tabs
./main.py:31:2: E117 over-indented
./main.py:32:1: W191 indentation contains tabs
./main.py:36:2: E117 over-indented
./main.py:44:1: W191 indentation contains tabs
./main.py:46:1: W191 indentation contains tabs
./main.py:47:1: W191 indentation contains tabs
./main.py:48:1: W191 indentation contains tabs
./main.py:49:1: W191 indentation contains tabs
./main.py:50:1: W191 indentation contains tabs
./main.py:51:1: W191 indentation contains tabs
./main.py:51:3: E117 over-indented
./main.py:52:1: W191 indentation contains tabs
./main.py:53:1: W191 indentation contains tabs
./main.py:54:1: W191 indentation contains tabs
./main.py:55:1: W191 indentation contains tabs
./main.py:67:1: W191 indentation contains tabs
./main.py:69:1: W191 indentation contains tabs
./main.py:70:1: W191 indentation contains tabs
./main.py:70:4: E117 over-indented
./main.py:71:1: W191 indentation contains tabs
./main.py:72:1: W191 indentation contains tabs
./main.py:73:1: W191 indentation contains tabs
./main.py:73:5: E117 over-indented
./main.py:74:1: W191 indentation contains tabs
./main.py:75:1: W191 indentation contains tabs
./main.py:76:1: W191 indentation contains tabs
./main.py:77:1: W191 indentation contains tabs
./main.py:77:4: E117 over-indented
./main.py:78:1: W191 indentation contains tabs
./main.py:79:1: W191 indentation contains tabs
./main.py:79:3: E117 over-indented
./main.py:83:1: E101 indentation contains mixed spaces and tabs
1     E101 indentation contains mixed spaces and tabs
12    E117 over-indented
36    W191 indentation contains tabs
```

--- a/main.py
+++ b/main.py
@@ -8,32 +8,32 @@ from datetime import datetime
 
 
 def parse_args():
-	parser = argparse.ArgumentParser(description="This script gets IP address information.")
-	parser.add_argument("ip", help="IP Address. Specify 'me' or an IP address.")
-	parser.add_argument("-o", "--output", dest='output', action='store_true', help="Write output of script to a file.")
-	return parser.parse_args()
+    parser = argparse.ArgumentParser(description="This script gets IP address information.")
+    parser.add_argument("ip", help="IP Address. Specify 'me' or an IP address.")
+    parser.add_argument("-o", "--output", dest='output', action='store_true', help="Write output of script to a file.")
+    return parser.parse_args()
 
 
 def get_external_ip(args):
-	if args.ip == 'me':
-		ip = get('https://api.ipify.org').text
-	else:
-		ip = args.ip
-	return ip
+    if args.ip == 'me':
+        ip = get('https://api.ipify.org').text
+    else:
+        ip = args.ip
+    return ip
 
 
 def get_ip_information(ip):
-	info = get(f'http://ip-api.com/json/{ip}')
-	return info.text
+    info = get(f'http://ip-api.com/json/{ip}')
+    return info.text
 
 
 def get_as_subnets(bgp_as):
-	r = get(f"https://api.hackertarget.com/aslookup/?q={bgp_as}").text
-	return r.split('"')[4].split("\n", 1)[1]
+    r = get(f"https://api.hackertarget.com/aslookup/?q={bgp_as}").text
+    return r.split('"')[4].split("\n", 1)[1]
 
 
 def main():
-	print("""
+    print("""
 ------------------------------
 |   IP Address Information   |
 |         Lookup Tool        |
@@ -41,42 +41,42 @@ def main():
 |     By Luke D. Tainton     |
 |        @luketainton        |
 ------------------------------
-	""")
+    """)
 
-	args = parse_args()
-	my_ip = get_external_ip(args)
-	my_info_raw = get_ip_information(my_ip)
-	my_info = json.loads(my_info_raw)
-	if my_info['status'] == "success":
-		location = f"{my_info['country']}/{my_info['regionName']}/{my_info['city']}"
-		timezone = my_info['timezone']
-		isp = my_info['isp']
-		bgp_as = my_info['as'].split(" ")[0]
-		subnets = get_as_subnets(bgp_as)
-		output = f"""
-IP Address:		{my_ip}
-Location:		{location}
-Timezone:		{timezone}
-ISP:			{isp}
-Autonomous System:	{bgp_as}
+    args = parse_args()
+    my_ip = get_external_ip(args)
+    my_info_raw = get_ip_information(my_ip)
+    my_info = json.loads(my_info_raw)
+    if my_info['status'] == "success":
+        location = f"{my_info['country']}/{my_info['regionName']}/{my_info['city']}"
+        timezone = my_info['timezone']
+        isp = my_info['isp']
+        bgp_as = my_info['as'].split(" ")[0]
+        subnets = get_as_subnets(bgp_as)
+        output = f"""
+IP Address:        {my_ip}
+Location:        {location}
+Timezone:        {timezone}
+ISP:            {isp}
+Autonomous System:    {bgp_as}
 
 Prefixes advertised by {bgp_as}:
 {subnets}
 
 Get more information at https://bgp.he.net/{bgp_as}.
-		"""
+        """
 
-		if args.output:
-			current_time = datetime.now().strftime('%Y%m%d-%H%M%S')
-			filename = f"output-{current_time}.txt"
-			with open(str(filename), 'w') as file:
-				file.write(output)
-				file.close()
-			print(f"Output has been written to {filename}.")
-		else:
-			print(output)
-	else:
-		print("The query failed. Please try again.")
+        if args.output:
+            current_time = datetime.now().strftime('%Y%m%d-%H%M%S')
+            filename = f"output-{current_time}.txt"
+            with open(str(filename), 'w') as file:
+                file.write(output)
+                file.close()
+            print(f"Output has been written to {filename}.")
+        else:
+            print(output)
+    else:
+        print("The query failed. Please try again.")
 
 
 if __name__ == "__main__":

